sdk/identity/Azure.Identity/README.md

```diff
@@ -202,12 +202,22 @@ var client = new SecretClient(new Uri("https://myvault.vault.azure.net/"), crede
 
 ## Sovereign cloud configuration
 
-By default, credentials authenticate to the Microsoft Entra endpoint for the Azure Public Cloud. To access resources in other clouds, such as Azure US Government or a private cloud, configure credentials with the `AuthorityHost` option. [AzureAuthorityHosts](https://learn.microsoft.com/dotnet/api/azure.identity.azureauthorityhosts?view=azure-dotnet) defines authorities for well-known clouds:
+By default, credentials authenticate to the Microsoft Entra endpoint for the Azure Public Cloud. To access resources in other clouds, such as Azure US Government or a private cloud, use one of the following solutions:
 
-```C# Snippet:AuthenticatingWithAuthorityHost
-var credential = new DefaultAzureCredential(new DefaultAzureCredentialOptions { AuthorityHost = AzureAuthorityHosts.AzureGovernment });
+1. Configure credentials with the [AuthorityHost](https://learn.microsoft.com/dotnet/api/azure.identity.tokencredentialoptions.authorityhost?view=azure-dotnet#azure-identity-tokencredentialoptions-authorityhost) property. For example:
+
+    ```C# Snippet:AuthenticatingWithAuthorityHost
+var credential = new DefaultAzureCredential(
+    new DefaultAzureCredentialOptions
+    {
+        AuthorityHost = AzureAuthorityHosts.AzureGovernment
+    });
 ```
 
+    [AzureAuthorityHosts](https://learn.microsoft.com/dotnet/api/azure.identity.azureauthorityhosts?view=azure-dotnet) defines authorities for well-known clouds:
```

Comment:
should we note that this setting takes effect for all credentials in the environment, rather than just the one specified?

Reply:
Yes, great idea